Demo/build.gradle

```diff
@@ -21,7 +21,6 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
 
         buildConfigField("String", "CLIENT_ID", paypalProperties["CLIENT_ID"] ?: "\"\"")
```

Comment:
We can remove this too now.

Reply:
@sshropshire the `clientId` is needed for the ElgibilityAPI. I think its a good practice to still have a `.properties` file with only the `CLIENT_ID`.

Reply:
Ok yeah I see the docs now. If only GraphQL could accept a client token instead of a `clientId` 🤔 .